### ISSUE TITLE
A11y: Video play button issues (focus indicator and SR description)

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/cgov-digital-platform#3900]
1. As a keyboard user, it is hard to discern when the play button of the video is in focus. Also, it only changes color which may not be sufficient for color blind users.  Add a focus ring indicator to the play button.

2. As a screen reader user, when encountering a video as it is currently implemented, the faux play button does not denote that it is a video and that clicking it will start a youtube video.  
  The play button is currently a div with a click listener, It should be converted to a button or at the very least, an aria `role=button`.  An aria-label should be applied, letting the screen reader user know that clicking on the interactive element will start a youtube video.   (This label will also have to be translated for use on spanish)
Note: A spanish translation for the new label will also be needed

> **ESTIMATE** TBD

### Steps to reproduce the issue
1. Navigate to a video component on the digital platform using keyboard tabbing
2. Notice that the play button turns from black to red, like the rollover

1.  Turn on screen reader
2. Navigate to a video component on the digital platform
3. Notice that it only calls out the title of the video

### Additional details / screenshot
- 
![Screenshot 2023-06-05 at 12 19 35 PM (1)](https://github.com/NCIOCPL/cgov-digital-platform/assets/45469809/7095db06-01b5-47a5-8f68-3bb82cafb355)
